library/src/main/AndroidManifest.xml

```diff
@@ -10,9 +10,9 @@
         android:largeHeap="true">
 
         <activity android:name=".activity.NaraePickerActivity"
-            android:theme="@style/Theme.AppCompat.Light.DarkActionBar"/>
+            android:theme="@style/NaraeImagePickerTheme"/>
```

Comment:
When applying a style in Activity by programmatically, you don't need ```NaraeImagePickerTheme``` and extending that style. I can produce the same effort by this statement.

```
<style name="CustomPickerTheme" parent="Theme.AppCompat">
        <item name="colorPrimary">@color/colorPrimary</item>
        <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
        <item name="colorAccent">@color/colorAccent</item>
    </style>
```

Reply:
I was add that theme for two reasons.
1. Refactor code
2. Guide user that what should be parent of their custom theme. (Strictly, not should be but for help users)
If you still fill this is not necessary, then I will remove it.

Reply:
@zeallat 

Although AppCompat is used heavily, recent [MaterialComponent](https://material.io/develop/android/docs/getting-started/) should use ```Theme.MaterialComponent``` theme rather than ```Theme.AppComapt```. If so, at least this library should provide a base style for:

- Theme.AppCompat.Light.DarkActionBar
- Theme.AppCompat
- Theme.AppCompat.DayNight
- Theme.MaterialComponents
- Theme.MaterialComponents.Light.DarkActionBar

And I think many apps already have their own styles defined. So that's no problem to use self-defined style rather than base style.

Reply:
@WindSekirun LGTM